### PR TITLE
Armanda profile

### PR DESCRIPTION
**Pull request by rachelcarmena**
Armanda is here!

### CODE REVIEW

**Comment by alvarogarcia7** [comment]
'Apprentice Software Craftswoman' or just 'Apprentice'?

**Comment by rachelcarmena** [comment]
Armanda agrees with you. I'm going to fix it ;)

File: _data/team.yml [hunk 1]
```diff
@@ -259,3 +259,14 @@
...
   github: charlyraffellini
+
+- id: armanda
+  name: Armanda du Plessis
+  image: /assets/img/custom/team/armanda_du_plessis.jpg
+  role: Apprentice
+  fullDescription: >
+    <p>Armanda has worked as a programmer, software designer, web developer and freelance consultant across a variety of industries, in both South Africa and the UK.</p>
+    <p>Her career has been a curious journey of constant learning and improvement. She has broad technical knowledge, is a creative thinker with an inquisitive mind and loves solving problems. </p>
+    <p>She is drawn to the passions and values of the software craftsman and fully believes in its enlightened culture.</p>
```
Comment (jukecraft) [inline _data/team.yml]: are you talking about the book "the software craftsman" or about software craftsmanship?

Comment (MissArmieJ) [reply _data/team.yml]: craftsmanship - thanks Franzi

Comment (rachelcarmena) [reply _data/team.yml]: Armanda agrees with you! Thanks @franziskas! Done!

**Pull request merged**